api/src/db/migrations/migrations.test.ts

```diff
@@ -137,8 +141,12 @@ describe("migrations", () => {
 
     it("runs all migrations and gets user to current version", () => {
       let user: unknown = generateV0UserData({});
-      for (const func of Migrations) {
-        user = func(user);
+      try {
+        for (const func of Migrations) {
+          user = func(user);
+        }
+      } catch (error) {
+        logger.LogError(`Dynamo User Migration Test Error - Error: ${error}`);
```

Comment:
If we could get this to output to the console that would be 👨‍🍳 💋

Reply:
I created a consoleLogWriter in the LogWriter file.. Please take a look